docs/source/conf.py

```diff
@@ -69,9 +69,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.12'
+version = '0.13'
 # The full version, including alpha/beta/rc tags.
-release = 'v0.12.1.post0.dev0'
+release = 'v0.13.0.post0.dev0'
```

Comment:
```suggestion
release = 'v0.13.0'
```

(as it will be a stable release)

Reply:
done